Darjusch/fitup
# Frontend

Frontend for the Fitup smartcontract:
* You can connect with your Metamask
* You can create a Bet
* You can get a Bet
* You can payout a Bet
To test:
1. Clone repo.
2. cd into root folder
3. npm install
4. npm deploy
5. copy Fitup.json from the build folder ( Fitup/build/contracts/Fitup.json ) into ( Fitup/client/src/abi/Fitup.json )
6. cd into client folder
7. npm install
8. npm start
9. click enable ethereum button
10. it asks you to connect to Metamask -> Approve
11. Open the dev console
12. Now you can click on Create Bet, Get Bet, Payout Bet and see the result in the console


Problems that might occur:

Currently we have an import problem and need to add "type":"module" into the package.json when we run npm start and take it out when we run truffle deploy.
